packages/FileFormatter/FileFormatter.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\FileFormatter;
+
+use Rector\Core\Configuration\Option;
+use Rector\Core\Contract\EditorConfig\EditorConfigParserInterface;
+use Rector\Core\Contract\Formatter\FileFormatterInterface;
+use Rector\Core\ValueObject\Application\File;
+use Symplify\PackageBuilder\Parameter\ParameterProvider;
+
+final class FileFormatter
+{
+    /**
+     * @var EditorConfigParserInterface
+     */
+    private $editorConfigParser;
+
+    /**
+     * @var FileFormatterInterface[]
+     */
+    private $fileFormatters;
+
+    /**
+     * @var ParameterProvider
+     */
+    private $parameterProvider;
+
+    /**
+     * @param FileFormatterInterface[] $fileFormatters
+     */
+    public function __construct(EditorConfigParserInterface $editorConfigParser, ParameterProvider $parameterProvider, array $fileFormatters = [])
+    {
+        $this->editorConfigParser = $editorConfigParser;
+        $this->fileFormatters = $fileFormatters;
+        $this->parameterProvider = $parameterProvider;
+    }
+
+    /**
+     * @param File[] $files
+     */
+    public function format(array $files): void
+    {
+        if(!$this->parameterProvider->provideBoolParameter(Option::ENABLE_EDITORCONFIG)) {
```

Comment:
Should be disabled by default at the beginning. Otherwise we are annoying some people at the beginning ;-)